src/promptflow/promptflow/_utils/tool_utils.py

```diff
@@ -111,7 +111,7 @@ def param_to_definition(param, gen_custom_type_conn=False) -> (InputDefinition,
 
 
 def function_to_interface(
-    f: Callable, initialize_inputs=None, gen_custom_type_conn=False, skip_prompt_template=False
+    f: Callable, initialize_inputs=None, gen_custom_type_conn=False, skip_prompt_template=False, add_ui_hints=True
```

Comment:
Why default value is `True`?

Reply:
Previously this pr affected all tool types, after discussion, this pr only affect custom-llm tool, changed the default value to False, thanks.